wiki/Ranking_Criteria/Ranking_Criteria_Council/en.md

```diff
@@ -0,0 +1,162 @@
+# Ranking Criteria Council
+
+The **Ranking Criteria Council** was a group of users tasked with reworking the [Ranking Criteria](/wiki/Ranking_Criteria) in 2016.
```

Comment:
make it a time span for clarity, something like "in 2016–2018", otherwise it sounds like they were only active in '16

Reply:
but they were only active in 2016, new group below mentions 2016-2018